sig-contributor-experience/README.md

```diff
@@ -49,7 +49,7 @@ The following subprojects are owned by sig-contributor-experience:
     - https://raw.githubusercontent.com/kubernetes/community/master/OWNERS
 - **community-management**
   - Owners:
-    - https://raw.githubusercontent.com/kubernetes/community/master/communication/OWNERS https://raw.githubusercontent.com/kubernetes/community/master/events/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/community/master/communication/OWNERS
```

Comment:
This is generated code using `sigs.yaml`. :)

You'll need to remove the line here:

https://github.com/kubernetes/community/blob/3bfd133d8624525326c62e809eb59c4eb0bcfb9b/sigs.yaml#L988

and then run `make generate`.

Reply:
Thanks. Fixed.